### PR TITLE
17 search results 2 link to see more results

### PR DESCRIPTION
**Pull request by kcockett**
### Description of changes
The Search Results page includes a link to "Show more results".  This adds a show page that displays all returned results, which is currently limited to 10 by our API key

### Screenshots (if applicable) of relevant views
Screenshot 2023-09-16 at 9 56 34 AM

### Test Suite
- [X] Are all tests within the test suite passing?
- [X] 100% coverage

#### Add GIF (NOT Optional)
Finally!

### CODE REVIEW

File: app/views/search_results/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<h2>Search Results for <%= @category %></h2>
+
+  <h3>Urgent Care results</h3>
```
Comment (z-fitch) [inline app/views/search_results/show.html.erb:3]: Should this be '@category results' so that its dynamic?

**Review by z-fitch — CHANGES_REQUESTED** [review]
Left a comment/question

**Comment by z-fitch** [comment]
Otherwise looks great!

**Review by garussell — APPROVED** [review]
This looks good.  I added all of the non-urgent categories in a separate branch.  I'm approving this and will merge.

**Review by z-fitch — APPROVED** [review]

**Pull request merged**